src/segmentation.py

```diff
@@ -5,7 +5,9 @@ def segment_image(img: np.ndarray):
     """
     Get the colors from an image using KMeans clustering
     :param img: The image to get the colors from
```

Comment:
Maybe be more specific about the input aswell. Those who are not familiar with the test may not know whats coming in as input. Also Specify the input comining in (already segemented from model or just cropped) may also be a good idea.

It may be helpful to also specifiy the color space thats coming in and out of this function.